docs/references/general/fitacf.md

```diff
@@ -29,7 +29,7 @@ Sometimes modes like `twofsound` will write data into a single file. In this cas
 RST includes several fitting algorithms which are used to determine the physical properties of the received backscatter (power, velocity, spectral width, elevation) from the mean autocorrelation and crosscorrelation function estimates. 
 RST includes the following fitting algorithms:
 
-- FITACF 2.5 (default)
+- FITACF 2.5 (optional)
```

Comment:
@ksterne I wonder if we can remove this entire section? This information is already available in more detail in the `make_fit` tutorial, and I think it belongs there. https://radar-software-toolkit-rst.readthedocs.io/en/release-rst-5.0/user_guide/make_fit/

Reply:
I've removed the hold `Fitting Algorithms` section now.  Is the `File Structure` a carryover from the RFCs?  I could see the intent of this section is to help defined some parameters and how the file is written, and read, from one record to the next.  It's true that it's not specific to fitacf files, I'm guessing its there since fitacf files are the general user level radar data file.  There's possibly another page as a tutorial or under SuperDARN data that could contain some of this more generic information about variables in the files.